examples/OpticalTracker/scripts/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# cmake wrapper
```

Comment:
We probably don't want to add a build wrapper to the dd4hep examples

Reply:
Removed (but added example `cmake` command to `README.md` for standalone build)